Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -128,6 +130,14 @@
 	end
 end
 
+% In Philips and dcm2niix20220720, often RepetitionTimePreparation is assigned the time right after the last readout and 
+% RepetitionTimeExcitation the length of the whole cycle - that needs to be fixed.
+if strcmpi(jsonOut.Manufacturer,'Philips')
+	if isfield(jsonOut,'RepetitionTimeExcitation') && jsonOut.RepetitionTimeExcitation >= jsonOut.RepetitionTimePreparation
```

Comment:
This is weird, we should choose one out of the two options:

Option 1. jsonOut == jsonIn and we run `rmfield` for each exception
Option 2. jsonOut == empty and we fill it per structure/field.

Now we use option 2 above for `RepetitionTimePreparation`and option 1 here?

Reply:
Not weird - DICOM, vendors, and dcm2nii behaves very weirdly. So we have to correct for their inconsistencies ad-hoc. Unfortunately. What you propose would only work if these people would stick to standards.

If a messy standard is on input and you want to provide a solid standard on output, you have to go for ad-hoc exceptions and fixes. The specific reason is given in the comment.

Reply:
OK, so `xASL_bids_BIDSifyASLJSON` merges `studyPar` and `jsonIn` to `jsonOut` right?

Can we then change it such that:
1) you always take `jsonOut` = `jsonIn` OR `studyPar` (NOT `jsonIn` = `jsonOut` or `jsonOut` = `jsonOut`)
2) merge `xASL_bids_BIDSifyASLJSON` and `xASL_bids_MergeStudyPar`

(Again I'm fine not doing that here but moving this to a new issue)

Reply:
Agree that the mixing of variables was confusing. I'm now merging studyPar+jsonIn to jsonInMerged. jsonIn and studyPar are used, but never modified. jsonInMerged is then cleaned, but we always write in the direction jsonInMerged->jsonOut.